packages/imperative/src/config/src/api/ConfigLayers.ts

```diff
@@ -49,11 +51,19 @@ export class ConfigLayers extends ConfigApi {
                 layer.properties = JSONC.parse(fileContents.toString()) as any;
                 layer.exists = true;
             } catch (e) {
-                throw new ImperativeError({
-                    msg: `Error parsing JSON in the file '${layer.path}'.\n` +
-                        `Please check this configuration file for errors.\nError details: ${e.message}\nLine ${e.line}, Column ${e.column}`,
-                    suppressDump: true
-                });
+                const msg = `Error parsing JSON in the file '${layer.path}'.\n` +
+                `Please check this configuration file for errors.\nError details: ${e.message}\nLine ${e.line}, Column ${e.column}`;
+                if (!opts?.ignoreErrors){
+                    throw new ImperativeError({
+                        msg:msg,
+                        suppressDump: true
+                    });
+                } else {
+                    const cmdResp: CommandResponse = new CommandResponse({
+                        responseFormat: "default"
+                    });
+                    cmdResp.console.log(TextUtils.chalk.red(msg));
```

Comment:
Should we use `Logger.getAppLogger()` here instead? I'm concerned that the `CommandResponse` option may not work well for extenders like Zowe Explorer running outside the context of a console.